src/coreclr/nativeaot/Runtime/windows/PalRedhawkMinWin.cpp

```diff
@@ -671,6 +671,42 @@ REDHAWK_PALEXPORT char* PalCopyTCharAsChar(const TCHAR* toCopy)
     return converted;
 }
 
+REDHAWK_PALEXPORT HANDLE PalLoadLibrary(const char* moduleName)
+{
+    if (moduleName == nullptr)
+    {
+        return 0;
+    }
+    size_t len = strlen(moduleName);
+    wchar_t* moduleNameWide = new (nothrow)wchar_t[len + 1];
+    if (moduleNameWide == nullptr)
+    {
+        return 0;
+    }
+    for (size_t i = 0; i < len; i++)
+    {
+        moduleNameWide[i] = (wchar_t)moduleName[i];
+    }
+    moduleNameWide[len] = '\0';
```

Comment:
This should use `MultiByteToWideChar` - the string being passed is UTF-8 and this "poor man's converter" will produce broken results if the file path to the executable or the string in `DOTNET_GCName` contain non-ascii characters.